xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -2170,11 +2105,7 @@ void CAMLCodec::SetVideoRect(const CRect &SrcRect, const CRect &DestRect)
   CRect gui, display;
   gui = CRect(0, 0, CDisplaySettings::GetInstance().GetCurrentResolutionInfo().iWidth, CDisplaySettings::GetInstance().GetCurrentResolutionInfo().iHeight);
 
-#ifdef TARGET_ANDROID
-  display = m_display_rect;
-#else
   display = gui;
-#endif
   if (gui != display)
```

Comment:
sorry for the late reply, I somehow missed your comment.

no it is not dead code. ref #11150